pennylane_lightning/core/_serialize.py

```diff
@@ -285,6 +285,9 @@ def _pauli_word(self, observable, wires_map: dict = None):
         def map_wire(wire: int):
             return wires_map[wire] if wires_map else wire
 
+        if len(observable) == 0:
+            return self.named_obs(PAULI_NAME_MAP["I"], [map_wire(0)])
```

Comment:
Does the `wires_map` map from (potentially custom) wire labels to state vector indices or the other way around?

Reply:
If the latter is true, then the following might be more correct
```suggestion
        if len(observable) == 0:
            return self.named_obs(PAULI_NAME_MAP["I"], [0])
```

Reply:
Nice one! Thanks @mudit2812 .